runtime/commonTest/src/kotlinx/serialization/protobuf/ProtobufMissingFieldsTest.kt

```diff
@@ -0,0 +1,138 @@
+package kotlinx.serialization.protobuf
+
+import kotlinx.serialization.Decoder
+import kotlinx.serialization.Encoder
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.SerialDescriptor
+import kotlinx.serialization.SerialId
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.internal.SerialClassDescImpl
+import kotlinx.serialization.json.JsonInput
+import kotlinx.serialization.json.JsonOutput
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFalse
+
+class ProtobufMissingFieldsTest {
+
+    private val buffer = byteArrayOf(10, 30, 8, 11, 16, 2, 26, 3, 115, 112, 97, 26, 6, 115, 112, 97, 95, 101, 115, 26, 2, 101, 115, 26, 5, 101, 115, 95, 101, 115, 32, 1, 16, 25)
+
+    @Test
+    fun deserialize() {
+        val items = ProtoBuf.load(Items.serializer(), buffer)
+        assertEquals(25, items.pageSize)
+        assertFalse(items.nextPage)
+        assertEquals(1, items.items.size)
+        assertEquals(ItemPlatform.Android, items.items[0].platform)
+        assertEquals(11, items.items[0].id)
+        assertEquals(listOf("spa", "spa_es", "es", "es_es"), items.items[0].language)
+        assertEquals(ItemContext.Context1, items.items[0].context)
+    }
+
+    @Test
+    fun deserializeWithoutFields() {
+        val items = ProtoBuf.load(ItemsWithoutFields.serializer(), buffer)
+        assertFalse(items.nextPage)
+        assertEquals(1, items.items.size)
+        assertEquals(11, items.items[0].id)
+        assertEquals(listOf("spa", "spa_es", "es", "es_es"), items.items[0].language)
+        assertEquals(ItemContext.Context1, items.items[0].context)
+    }
+}
+
+
+enum class ItemPlatform {
+    Unknown,
+    iOS,
+    Android
+}
+
+enum class ItemContext {
+    Unknown,
+    Context1,
+    Context2
+}
+
+@Serializable
+data class Items(
+    @SerialId(1)
+    val items: List<Item> = emptyList(),
+    @SerialId(2)
+    val pageSize: Int? = null,
+    @SerialId(3)
+    val nextPage: Boolean = false
+)
+
+@Serializable
+data class Item(
+    @SerialId(1)
+    val id: Int,
+    @SerialId(2) @Serializable(with = ItemPlatformSerializer::class)
+    val platform: ItemPlatform = ItemPlatform.Unknown,
+    @SerialId(3)
+    val language: List<String> = emptyList(),
+    @SerialId(4) @Serializable(with = ItemContextSerializer::class)
+    val context: ItemContext = ItemContext.Unknown
+)
+
+@Serializable
+data class ItemsWithoutFields(
```

Comment:
`ItemsWithoutPageSize` sounds more precise to me

Reply:
Sounds good.